### PR TITLE
Old and new express checkout flows

### PR DESCRIPTION
**Pull request by cloveras**

### CODE REVIEW

File: vipps-ecom-api.md [hunk 1]
```diff
@@ -214,6 +216,44 @@ To perform an express checkout, the merchant needs to send
...
+
+In the old flow for the express checkout it was easy for a user to overlook
+that the correct address and shipping method was used, as both were simply
+displayed on the payment confirmation page. The user did not have to actively
+choose and address and shipping method. This resulted in some users
+inadvertently confirming the wrong address and possibly also selecting the
+wrong shipping method. To fix this, we made it mandatory to select and address
+and shipping method. This was released as a minor update of the Vipps app.
+
+That worked well for all merchants that used Vipps Hurtigkasse _as intended_,
```
Comment (tht13) [inline vipps-ecom-api.md:231]: Don't think we need to go into this detail
Just say we added an optional longer flow where merchants can choose if this is appropriate

File: vipps-ecom-api.md [hunk 2]
```diff
@@ -214,6 +216,44 @@ To perform an express checkout, the merchant needs to send
...
+So: We changed it back to the way it was before, and now require merchants to
+explicitly specify the new express checkout flow.
+
+#### How to specify the old or new express checkout flow
+
+The old (and for some: "normal") express checkout flow is the default.
+You do not have to make any changes other than to specify
+`"paymentType": "eComm Express Payment"`.
+
+To get the new express checkout flow: Specify this in addition:
```
Comment (tht13) [inline vipps-ecom-api.md:249]: The field needs to be in the transaction object, specify something like this?
`Specify this field in the "transaction" object of the initiation request in addition"

File: vipps-ecom-api.md [hunk 3]
```diff
@@ -214,6 +216,44 @@ To perform an express checkout, the merchant needs to send
...
 support the `shippingDetails` and `consent` endpoints.
 
+### Old and new express checkout flow
+
+This does require some background info - please bear with us.
+
+In the old flow for the express checkout it was easy for a user to overlook
+that the correct address and shipping method was used, as both were simply
+displayed on the payment confirmation page. The user did not have to actively
+choose and address and shipping method. This resulted in some users
```
Comment (aleksen) [inline vipps-ecom-api.md:226]: an address

File: vipps-ecom-api.md (hunk 1 above)
Comment (cloveras) [reply vipps-ecom-api.md:231]: There are several thousand users out there, so _some_ background info is required. May shorten it a bit later.

File: vipps-ecom-api.md (hunk 2 above)
Comment (cloveras) [reply vipps-ecom-api.md:249]: Good point!

**Pull request merged**